src/dom.js

```diff
@@ -0,0 +1,75 @@
+import { signalFn } from "./signal";
+
+/**
+ * The dom module helps to connect view functions with signal circuits. A view
+ * can be mounted by passing a function that returns a views representation.
+ * mount/unmount will then take care of the signal creation and lifetime
+ * management for you. Also, mount/unmount helps in doing the necessary
+ * side-effects to the DOM by calling the provided `patchFn` (mount) and
+ * `cleanupFn` (unmount). The goal is to provide a simple interface consisting
+ * of these two functions but perform optimized rendering based on signals.
+ *
+ * @module dom
+ */
+
+/**
+ * Mounts the result of `viewFn` as a replacement of `root`.
```

Comment:
> ...as a replacement of `root`.

Yes. I think this is well defined and it should perhaps be the default mount-behaviour. However, it does not yet provide a clear method for mounting components dynamically. We should perhaps approach that problem from a use-case perspective though.

This is good.

Reply:
You're right, mounting a view as a child of a root is not supported (natively). I still have this at the back of my mind, but want to see where this is going with the provided `mount`. Of course, a flyps user can always wrap `mount` to do mount-as-child, so there is no limitation. In the future, flyps might provide this as a feature.